frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -210,6 +210,10 @@ action.GitMachete.FastForwardMergeSelectedToParentAction.description=Fast-forwar
 action.GitMachete.SyncSelectedToParentByMergeAction.text=Git Machete: Checkout and Sync to Parent by Merge
 action.GitMachete.SyncSelectedToParentByMergeAction.GitMacheteContextMenu.text=Checkout and Sync to Parent by _Merge
 action.GitMachete.SyncSelectedToParentByMergeAction.description=Checkout this branch and merge parent into it
+action.GitMachete.SyncSelectedToParentByMergeAction.warning-dialog.title=Merge operation warning
+action.GitMachete.SyncSelectedToParentByMergeAction.warning-dialog.message=Using merge to sync branches is discouraged, especially with stacked PRs. Ignore this message and proceed if this is a hotfix branch
```

Comment:
Hmmm wrt. _Ignore this message and proceed if this is a hotfix branch_... note that the case mentioned in https://slides.com/plipski/git-machete/#/12, doesn't tell about **hotfix branch** getting merged into anything/anything getting merged into hotfix branch. It's **hotfixed master** that's getting merged into develop! Pls clarify, or better just skip the second sentence completely to avoid confusing the 💩 out of users ;D